### DISCUSSION TITLE
Great App! Suggestion...

### DISCUSSION THREAD

**Discussion by hazlema** [rbreu/beeref#65] [Ideas & General Discussion]
Allow exporting of images that are in the beeref window.

I cut and paste, copy the image from the browser and paste it in beeref.  
I wanted to grab some color info from the image and needed to save the image as a file.  

This would be a great feature!

**Comment by NameUserIII** [comment]
You could use program that can grab colors from any part of screen like: "bella", "color picker" etc.
